### ISSUE TITLE
Valet should include a Notice we can include

### ISSUE THREAD

**Issue by steplercamp** [square/Valet#29]
Valet is awesome, but there doesn't appear to be a license we can use. Is Valet public domain? The License doesn't specify Square anywhere.

**Comment by dfed** [comment]
Valet is licensed under [Apache 2.0](https://github.com/square/Valet/blob/master/LICENSE) by Square. Please reopen if you have other licensing questions.

**Comment by steplercamp** [comment]
Sorry for the inconvenience, I should have been more clear. 

Can you please fill out the boiler plate? I'd submit a PR but it seems presumptuous of me. 

https://github.com/square/Valet/blob/master/LICENSE#L190

**Comment by dfed** [comment]
That boilerplate is supposed to be left blank in the LICENSE file. The copyright you can find at the top of every .h or .m file: `Copyright 2015 Square, Inc.`

**Comment by steplercamp** [comment]
Whoops, now I feel extra extra silly. Sorry for all the hassle over nothing. 

Thanks!

Hopefully these puppies make up for some of the trouble. 
http://i.imgur.com/nUokjAv.gif

**Comment by dfed** [comment]
No worries! Thanks for your interest in Valet, and for the puppies :)

**Comment by dcow** [comment]
@dfed Square lets you ship these libraries without requiring attribution in binary form? That's nice!

**Comment by dfed** [comment]
@dcow per the Apache 2.0 license, attribution is required.

http://www.apache.org/foundation/license-faq.html

**Comment by dcow** [comment]
@dfed read section 4 notably subsection c and d of the Apache license. The only attribution required for **binary** distributions is reproduction of the NOTICE file in its entirety.

**Comment by dfed** [comment]
@dcow let me rephrase the above: I'm not a lawyer. Consult your lawyer before adopting open source libraries :)

**Comment by dcow** [comment]
@dfed of course. I'm simply surprised that the Valet project does not include a notice file because it means my Derivative Works that are not in source form do not need to include any attribution to Square. If this is intended by you, fine. But based on your comments above, this seems like news.
